CGPs/cgp-0134.md

```diff
@@ -0,0 +1,108 @@
+ 
+## Background Information
+The chain was launched with 600m Celo at genesis in April 2020. The remaining 400m were reserved for epoch rewards. These epoch rewards are currently going to voting rewards, validator rewards, community fund and carbon offset fund. 
+
+If we look at the circulating supply, we have a total of 690m Celo so that means that the chain has created new 90m Celo since genesis. This means our annual inflation rate has been less than 3.6% and current network inflation rate is 2.09%. There are about 40k new Celo minted daily. By looking at the code, there was supposed to be a linear reward of 200m over 15 years so that’s about 36.5k Celo per epoch and 13.3m Celo per year. 
+
+As per the initial token release schedule, the community fund would have received 171m Celo over the span of 30 years. There was an assumption that if there are low voting rewards then people would not lock their Celo and this could be a security risk but reality is that people have been rather insensitive to the voting rewards and very few people know the current rewards. The community fund is set to receive a total of an extra 120m Celo from the Mento reserve.
+
+## Proposed Changes
+ 
+This proposal is to revert back and freeze the reward multiplier to its previous value two years ago. The following changes will be implemented:
+- Reward multiplier will be freezed and re-adjusted to 0.78. This will stabilize validators rewards around $58.5k
+- To account for the total of 150m Celo being transferred from Mento, this proposal will significally cut down the emissions to the community fund
+- We will freeze the emission to locked voting Celo at its current rate. This will have the advantage of maintaining consistent yield and prevent further decreased that has occurend in the past for the locked voting Celo
+
+
+```
+[
+    {
+      "contract": "EpochRewards",
+      "function": "setRewardsMultiplierParameters",
+      "args": [
+        "2000000000000000000000000",
+        "0",
+        "0"
+      ],
+      "value": "0"
+    },
+    {
+        "contract": "EpochRewards",
+        "function": "setTargetVotingYield",
+        "args": [
+            "101500000000000000000"
+        ],
+        "value": "0"
+    },
+    {
+        "contract": "EpochRewards",
+        "function": "setTargetVotingYieldParameters",
+        "args": [
+            "500000000000000000000",
+            "0"
+        ],
+        "value": "0"
+    },
+    {
+        "contract": "EpochRewards",
+        "function": "setCommunityRewardFraction",
+        "args": [
+            "10000000000000000000000"
+        ],
+        "value": "0"
+    }
+]
+```
+
+These changes have been tested on Alfajores testnet 
```

Comment:
Not exactly these ones.

Reply:
Thanks for the clarity. Have asked @martinvol  to weigh in on the actual parameters that were used.

Reply:
Is this solved? @0xzoz @rcroessmann

Reply:
Yes, Dee and Pavel tested them on Alfajores.